### PR TITLE
Define what a notification is

### PR DESCRIPTION
**Pull request by thompiler**
Add section that explains what a notification can refer to.

## Screenshot

screen shot 2018-03-05 at 4 48 00 pm

### CODE REVIEW

**Comment by phumpal** [comment]
You beat me to it! @thompiler rebase?

File: jekyll/_docs/airbrake-faq/what-triggers-a-new-email.md [hunk 1]
```diff
@@ -16,17 +15,24 @@ We currently review...
...
 - The file in which the error occurred
 - The line number at which the error occurred
-- The RAILS_ENV that was set when the error occurred
+- The `RAILS_ENV` that was set when the error occurred
+
+## What is a notification?
+A notification can be:
+
+- Your personal email notifications
+- Your team's messaging (eg: Slack or HipChat) notifications
```
Comment (kyrylo) [inline jekyll/_docs/airbrake-faq/what-triggers-a-new-email.md]: I think HipChat was renamed to something, so we could either use the new name or mention another integration.

**Comment by thompiler** [comment]
>You beat me to it! @thompiler rebase?

Yup! Just pushed the rebased version. </img>

File: jekyll/_docs/airbrake-faq/what-triggers-a-new-email.md (hunk 1 above)
Comment (thompiler) [reply jekyll/_docs/airbrake-faq/what-triggers-a-new-email.md]: Oh yeah I think their new app is Stride: https://www.stride.com/

I think it's a different API though so we shouldn't just switch the name out. I'll just remove it.

**Pull request merged**